token-locker/src/lib.rs

```diff
@@ -68,54 +73,80 @@ pub struct Contract {
     pub is_whitelist_mode_enabled: bool,
```

Comment:
The Ethereum Factory also have a whitelist. Are these whitelists consistent with each other? If yes, how is this achieved?